pkg/localize/locales/en/cmd/cluster.en.toml

```diff
@@ -301,11 +305,21 @@ Client ID:     {{.ClientID}}
 Make a copy of the client ID to store in a safe place. Credentials won't appear again after closing the terminal.
 
 You will need to assign permissions to service account in order to use it. 
-For example for Kafka service you should execute the following command to grant access to the service account:
+'''
+
+[cluster.kubernetes.printKafkaAccessCommands]
+one = '''
+For Kafka service you should execute the following command to grant access to the service account:
 
   $ rhoas kafka acl grant-access --producer --consumer --service-account {{.ClientID}} --topic all --group all
 '''
- 
+
+[cluster.kubernetes.printRegistryAccessCommands]
+one = '''
+For service registry you should execute the following command to grant access to the service account:
```

Comment:
Same as above. 
For registry we do not grant access but `assign new role`. Registry by default will be read only. We need to mention that so if you only need read only access you do not need to use this command